glue.tf

```diff
@@ -0,0 +1,103 @@
+resource "aws_glue_catalog_database" "default" {
+  name = aws_rds_cluster.default.cluster_identifier
+}
+
+resource "aws_glue_connection" "default" {
+  connection_properties = {
+    JDBC_CONNECTION_URL = "jdbc:${var.engine}://${aws_rds_cluster.default.endpoint}/${aws_glue_catalog_database.default.name}"
+    PASSWORD            = random_string.random_dbpassword.result
+    USERNAME            = var.username
+  }
+
+  name = "glue-connection"
+
+  physical_connection_requirements {
+    availability_zone      = aws_subnet.private.0.availability_zone
+    security_group_id_list =["aws_security_group.base_sg.id",
+    ]
+    subnet_id              = aws_subnet.private.0.id
+  }
+}
+
+
+
+resource "aws_glue_crawler" "default" {
+  configuration = <<CONFIGURATION
+  {
+   "Version": 1.0,
+   "CrawlerOutput": {
+    "Partitions": {
+      "AddOrUpdateBehavior": "InheritFromTable"
+    },
+    "Tables": {
+      "AddOrUpdateBehavior": "MergeNewColumns"
+    }
+   }
+  }
+  
+CONFIGURATION
+  database_name = aws_glue_catalog_database.default.name
+  name          = "glue-crawler"
```

Comment:
use name prefix. Or else when we cannot use this module more than once, as the resources have same name.